CHANGELOG.md

```diff
@@ -5,13 +5,21 @@ Full documentation for hipCUB is available at [https://rocm.docs.amd.com/project
 ## (Unreleased) hipCUB-x.x.x for ROCm 6.4.0
 
 ### Added
+* Added regression tests to `rtest.py`. Regression tests are a subset of tests that caused hardware problems for past emulation environments.
+  * Can be run with `python rtest.py [--emulation|-e|--test|-t]=regression`
```

Comment:
If you combine with the point above, you can remove this line.

Reply:
To keep things consistent with other changelogs for other primitive library, I think we should keep it as two lines.